status/samples.py

```diff
@@ -0,0 +1,231 @@
+"""Handlers related with samples
+"""
+
+import tornado.web
+import json
+import cStringIO
+
+import matplotlib.gridspec as gridspec
```

Comment:
What are these files? Are they meant to be here?

Reply:
I accidentally removed the samples.py file, and then just put it back (the whole file without having edited it) so I have not even looked at those lines, but I will check..